dag/dag.go

```diff
@@ -33,6 +36,7 @@ func (dag *DAG) GenerateDAG(buildPath string, registryPrefix string) {
 			if err != nil {
 				return err
 			}
+			dckfile.ContextRelativePath = strings.ReplaceAll(filePath, workingDir+"/", "")
```

Comment:
If the relative path is needed in the `dgoss` test runner, why don't you compute it when it is needed instead of using the `dockerfile.Dockerfile` struct to pass the information ? 
You could pass the `workingDir` in the `dgoss.TestRunner` struct, I think it is cleaner this way, and no need to add the field to the `dockerfile.Dockerfile`.
We should avoid mixing concepts so each package remain decoupled from others.